src/main/java/view/impl/UsersMenu.java

```diff
@@ -0,0 +1,71 @@
+package view.impl;
+
+import dao.UserDao;
+import model.User;
+import service.UserService;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+
+
+public class UsersMenu {
```

Comment:
why users not user?

Reply:
I got this name from project task. I think it because we already have **User menus** in another part.